.github/workflows/quality.yml

```diff
@@ -113,13 +113,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Dependency analysis
-        run: ./gradlew buildHealth $CI_GRADLE_ARG_PROPERTIES
+        run: ./gradlew dependencyCheckAnalyze $CI_GRADLE_ARG_PROPERTIES
```

Comment:
Out of curiosity, what other things (other than checking dependencies) does the `buildHealth` task? I could not find any doc on it.

Reply:
This is not really clear to me as well. It comes from https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin#add-to-your-project-and-use . Maybe @ofalvai have a better visibility on that (see https://github.com/vector-im/element-android/pull/5780#discussion_r876989692).